### PR TITLE
k8s advertised host fix

### PR DESCRIPTION
**Pull request by maikschwan**
This PR enables to overwrite the advertised host settings in the `server.properties` file of kafka via env vars

### CODE REVIEW

**Comment by maikschwan** [comment]
This PR depends on https://github.com/leanix/leanix-deploy/pull/327

File: kafka/scripts/start-kafka.sh [hunk 1]
```diff
@@ -14,6 +14,12 @@ if [ ! -z "$HELIOS_PORT_kafka" ]; then
     ADVERTISED_PORT=`echo $HELIOS_PORT_kafka | cut -d':' -f 2`
 fi
 
+# Mounted kubernetes configMap values are read only. Since we need to write to the server.properties file
+# we mount the data to /tmp and then copy it to the target folder to be modified. https://github.com/kubernetes/kubernetes/issues/62099
+echo "Copying /tmp/server.properties to ${KAFKA_HOME}/config/server.properties"
```
Comment (set-leanix) [inline kafka/scripts/start-kafka.sh]: You could also do `cp -v` instead.

File: kafka/dockerbuild.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+COMMIT_HASH=$(git rev-parse --short HEAD)
```
Comment (set-leanix) [inline kafka/dockerbuild.sh]: Requires you to be in that folder. How about 

```
cd $( cd "$( dirname "$0" )"; pwd )
```

to change to the script's directory.

**Review by set-leanix — CHANGES_REQUESTED** [review]

File: kafka/dockerbuild.sh (hunk 2 above)
Comment (maikschwan) [reply kafka/dockerbuild.sh]: I was just thinking about remove the whole file actually. If I got this right then there is a build trigger on that repo that builds the docker image with considering the `dockerbuild.sh`. What do you think?

**Review by set-leanix — APPROVED** [review]
✅

**Pull request merged**